test/e2e/journeys/datesCantAttendDatePicker.test.js

```diff
@@ -0,0 +1,60 @@
+const doYouWantTextMsgRemindersContent = require('steps/sms-notify/text-reminders/content.en.json');
+const selectors = require('steps/check-your-appeal/selectors');
+const DateUtils = require('utils/DateUtils');
+const moment = require('moment');
+const paths = require('paths');
+
+const doYouWantTextMsgReminders = doYouWantTextMsgRemindersContent.fields.doYouWantTextMsgReminders;
+const datesYouCantAttend = selectors.theHearing.datesYouCantAttend;
+const datesYouCantAttendHearingAnswer = `${datesYouCantAttend}  ${selectors.answer}`;
+const datesYouCantAttendHearingChange = `${datesYouCantAttend}  ${selectors.change}`;
+
+Feature('Appellant PIP, one month ago, attends hearing with dates cannot attend using date-picker');
+
+Before(I => {
+  I.createTheSession();
+  I.seeCurrentUrlEquals(paths.start.benefitType);
+});
+
+After(I => {
+  I.endTheSession();
+});
+
+Scenario('Selects date of when they cannot attend the hearing', async I => {
+  const randomWeekDay = DateUtils.getDateInMilliseconds(
+    DateUtils.getRandomWeekDayFromDate(moment().startOf('day').add(5, 'weeks'))
```

Comment:
@paulgain The way I could only think about selecting a date in the date-picker is by using their `data-date` attribute which is a date in milliseconds e.g. (5th July = 1530748800000). So it needs to be the start of the day to match the date otherwise it won't find the element

Reply:
Take a look at this: https://github.com/hmcts/track-your-appeal-frontend/blob/master/app/core/tyaNunjucks.js
This will help you work with BST.